### DISCUSSION TITLE
[Enhancement] support webstrom

### DISCUSSION THREAD

**Discussion by mivui** [element-plus/element-plus#8630] [Feature Request]
Feature type: **`Enhancement`**

## Details

- Enhanced component name: `All`

## Description

https://github.com/JetBrains/web-types

### Additional comments

_(empty)_

**Comment by tolking** [comment]
We already provide [web-types.json](https://unpkg.com/browse/element-plus@2.2.8/web-types.json) file.

I remember this file was based on version 1.0. I do not use the webstrom, so I do not know the current compatibility of this file.

For the webstrom we are currently only able to provide some basic information.

It is recommended to use VSCode + Volar, that more accurate auxiliary support can be provided through the Ts type.
